test/Libraries/DynamoMSOfficeTests/DynamoMSOfficeTests.csproj

```diff
@@ -33,7 +33,7 @@
         </ProjectReference>
     </ItemGroup>
   <ItemGroup>
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" ExcludeAssets="none" />
+      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2"/>
```

Comment:
@QilongTang sorry I'm confused, so you made this change because DynamoCoreTests.csproj already includes the assets?

Reply:
Exactly, `DynamoCoreTests.csproj` already includes the assets so there is no need for other tests to be included. In the future, we probably want a sln with all the test projects excluded for any CoreRuntime build. This is a bit harder to maintain but saves a ton for later stages.